src/frontend/context/SøknadContext.ts

```diff
@@ -33,14 +37,63 @@ const [SøknadProvider, useSøknad] = createUseContext(
         } = useBehandling();
         const { fagsakId } = useSakOgBehandlingParams();
         const navigate = useNavigate();
-        const { bruker, minimalFagsak } = useFagsakContext();
-        const [visBekreftModal, settVisBekreftModal] = React.useState<boolean>(false);
+        const {
+            bruker,
+            minimalFagsak,
+            skjemaHarValgtFortroligBarn,
+            settSkjemaHarValgtFortroligBarn,
+        } = useFagsakContext();
+        const [visBekreftModal, settVisBekreftModal] = useState<boolean>(false);
 
         const barnMedLøpendeUtbetaling =
             minimalFagsak.status === RessursStatus.SUKSESS
                 ? hentBarnMedLøpendeUtbetaling(minimalFagsak.data)
                 : new Set();
 
+        const brukerData = hentDataFraRessurs(bruker);
+        const erBrukerFortrolig =
```

Comment:
For å være helt presis burde alle variabelnavn med `*Fortrolig` egentlig vært renamet til `*StrengtFortrolig`, siden kun fortrolig (aka kode 7) er noe annet en strengt fortrolig (aka kode 6)

Reply:
Endret relevante funksjoner og variabler fra ..Fortrolig... Til ...StrengtFortrolig..